File: Rregulloje.UI/RregullojeWebAPI/src/app/_ViewModels/IssueViewModel.ts
export class IssueViewModel {
  id: number;
  name: string;
  lastName: string;
  phonenumber: string;
  email: string;
  city: string;
  livingobject: string;
  floor: string;
  livingentrynumber: string;
  apartmentnumber: string;
  issuesubject: string;
  message: string;

  constructor(id: number = null,
    name: string = null,
    lastName: string = null,
    phonenumber: string = null,
    email: string = null,
    city: string = null,
    livingobject: string = null,
    floor: string = null,
    livingentrynumber: string = null,
    apartmentnumber: string = null,
    issuesubject: string = null,
    message: string = null) {
    this.id = id;
    this.name = name;
    this.lastName = lastName;
    this.phonenumber = phonenumber;
    this.email = email;
    this.city = city;
    this.livingobject = livingobject;
    this.floor = floor;
    this.livingentrynumber = livingentrynumber;
    this.apartmentnumber = apartmentnumber;
    this.issuesubject = issuesubject;
    this.message = message;
  }
}

File: Rregulloje.Application/ViewModels/IssueViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Rregulloje.Application.ViewModels
{
    public class IssueViewModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string ToEmail { get; set; }
        public string City { get; set; }
        public string LivingObject { get; set; }
        public string Floor { get; set; }
        public string LivingEntryNumber { get; set; }
        public string AppartmentNumber { get; set; }
        public string IssueSubject { get; set; }
        public string Message { get; set; }
    }
}
